css/quicklook.css

```diff
@@ -122,7 +122,8 @@
 }
 .elfinder-quicklook-titlebar-icon .ui-icon {
 	position: relative;
-	margin: -9px 3px 0px 0px;
+	margin: 0px 3px 0px 0px;
+	display: inline-block;
```

Comment:
What kind of problem are you having? Is there a screenshot of the problem?

Reply:
When I checked, the display position shifts downward due to this change.

![180105-002248](https://user-images.githubusercontent.com/1412630/34570395-028bb622-f1af-11e7-8862-d09c3a351e55.png)

Reply:
If your amendment is valid for plain elFinder, please make another branch and send it as a separate pull request.